.github/workflows/ci.yml

```diff
@@ -37,31 +37,32 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.6', '3.9']
+        python-version: ['3.7', '3.9']
         use-conda: ['Yes', 'No']
         include:
         - os: ubuntu-latest
           special-invocation: 'xvfb-run --auto-servernum '
-        - python-version: '3.6'
+        - python-version: '3.7'
           use-conda: 'No'
-          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.6 defaults channel
+          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.7 defaults channel
           pyside2-version: 5.12.0  # 5.12.1-5.12.6 fails on collection/segfaults on patch test
         - os: ubuntu-latest
-          python-version: '3.6'
+          python-version: '3.7'
           use-conda: 'No'
           skip-pyqt6: true  # No wheels on Py 3.6 Linux CIs
```

Comment:
This skip (and thus this whole section) can likely can be removed, unless the issue also persists on the 3.7 wheels